src/header.tsx

```diff
@@ -93,15 +94,7 @@ export const Header: React.FC<HeaderProps> = ({
                 <Box paddingRight={16}>
                     <Stack space={8}>
                         {pretitle && renderRichText(pretitle, {color: vars.colors.textPrimary})}
-                        {small ? (
-                            <Text4 regular role="heading" aria-level={2}>
-                                {title}
-                            </Text4>
-                        ) : (
-                            <Text6 role="heading" aria-level={2}>
-                                {title}
-                            </Text6>
-                        )}
+                        {small ? <Title2>{title}</Title2> : <Title3>{title}</Title3>}
```

Comment:
To keep the heading level we should use `as="h2"` here
```suggestion
                        {small ? <Title2 as="h2">{title}</Title2> : <Title3 as="h2">{title}</Title3>}
```

Reply:
fixed